UniqueFilenameClass/Arguments.cs

```diff
@@ -0,0 +1,29 @@
+﻿using MyConsoleApp.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyConsoleApp.UniqueFilenameClass
+{
+    public class Arguments : IArguments
+    {
+        public List<ArgumentsDto> ParseArguments(string[] args)
+        {
+            var argumentsList = new List<ArgumentsDto>();
```

Comment:
My tip here is that I almost always use the variable name "result" for the variable that will be returned from the method. I also declare that line as the very first one in my method.